pebbles/static/js/controllers/DashboardController.js

```diff
@@ -305,8 +309,11 @@ app.controller('DashboardController', ['$q', '$scope', '$routeParams', '$interva
         $scope.startPolling();
     }]);
 
-app.controller('ModalShowPasswordController', function($scope, $modalInstance, instance) {
+app.controller('ModalShowPasswordController', function($scope, $modalInstance, instance, blueprint) {
     $scope.instance = instance
+    if(blueprint[0].template_name == 'Rstudio') {
```

Comment:
Am I right in thinking this introduces an implied requirement that Rstudio template name must equal '"Rstudio"? There's no overriding of the '==' operator or anything to make that e.g. a case insensitive .contains() comparison that would be more generic?

Also, such a functionality should be mentioned somewhere in the documentation.